scipy/signal/spectral.py

```diff
@@ -697,36 +1262,35 @@ def _spectral_helper(x, y, fs=1.0, window='hann', nperseg=256,
     References
     ----------
     .. [1] Stack Overflow, "Rolling window for 1D arrays in Numpy?",
-        http://stackoverflow.com/a/6811241
-    .. [2] Stack Overflow, "Using strides for an efficient moving average
-        filter", http://stackoverflow.com/a/4947453
+           http://stackoverflow.com/a/6811241
+    .. [2] Stack Overflow, "Using strides for an efficient moving
+           average filter", http://stackoverflow.com/a/4947453
 
     Notes
     -----
     Adapted from matplotlib.mlab
 
     .. versionadded:: 0.16.0
     """
-    if mode not in ['psd', 'complex', 'magnitude', 'angle', 'phase']:
```

Comment:
there was a discussion and issue somewhere about these modes being useful for some case. is that case now covered by ``stft``? And doesn't this need a deprecation (could be handled by a conversion elsewhere in this PR, if so I haven't found it yet)?